src/SqlUtils.ts

```diff
@@ -116,40 +116,6 @@ export default class SqlUtils {
         return ipAddress;
     }
 
-    /**
-     * Opens a new connection to the database and executes a T-SQL script on it.
-     * @param connectionConfig Config object for the connection
-     * @param command T-SQL to be executed on the connection
-     */
-    static async executeSql(connectionConfig: SqlConnectionConfig, command: string): Promise<void> {
-        let pool: mssql.ConnectionPool | undefined;
-        try {
-            pool = await mssql.connect(connectionConfig.Config);
-            const result = await pool.query(command);
-            
-            // Display result
-            for (let i = 0; i < result.recordsets.length; i++) {
-                console.log(`Rows affected: ${result.rowsAffected[i]}`);
-                // Displays query result as JSON string. Future improvement: IRecordSet has function toTable() 
-                // which returns more metadata about the query results that can be used to build a cleaner output
-                console.log(`Result: ${JSON.stringify(result.recordsets[i])}`);
```

Comment:
>  console.log(`Result: ${JSON.stringify(result.recordsets[i])}`); [](http://example.com/codeflow?start=14&length=65)

is it ok that moving to sqlcmd's output format is a breaking change for users of this task?

Reply:
Prior to #99 we were using sqlcmd, so this will actually bring us back to parity with the old format :)